src/Paramore.Brighter/CommandProcessorBuilder.cs

```diff
@@ -245,7 +247,8 @@ public CommandProcessor Build()
                     outboxTimeout: _outboxWriteTimeout,
                     featureSwitchRegistry: _featureSwitchRegistry,
                     boxTransactionConnectionProvider: _overridingBoxTransactionConnectionProvider,
-                    outboxBulkChunkSize: _outboxBulkChunkSize
+                    outboxBulkChunkSize: _outboxBulkChunkSize,
+                    messageTransformerFactory: _transformerFactory
```

Comment:
Does this mean that we only want this to work with External Bus without Request Reply Queues?

Reply:
Not the intent.